core/storage/merkle/trie_db.hpp

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_MERKLE_TRIE_HPP
+#define KAGOME_MERKLE_TRIE_HPP
+
+#include "common/buffer.hpp"
+#include "storage/keyvalue.hpp"
+
+namespace kagome::storage::merkle {
+
+  class TrieDb : public KeyValue {
```

Comment:
Also add short description of this class

Reply:
Done